contracts/core/Controller.sol

```diff
@@ -274,6 +272,16 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         _;
     }
 
+    /**
+     * @notice modifier to check if the sender is the liquidation manager address
+     */
+    modifier onlyLiquidationManager() {
+        if (addressbook.getLiquidationManager() != address(0)) {
+            require(msg.sender == addressbook.getLiquidationManager(), "C38");
```

Comment:
for cleanliness:

require(msg.sender == addressbook.getLiquidationManager() || addressbook.getLiquidationManager() == address(0), "C38");

Reply:
https://github.com/ribbon-finance/GammaProtocol-OTC/pull/2/commits/43d8fae36603a28412edcdebe2cbf58fd72eac9c